mesheryctl/cmd/stop.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 var (
-	resetFlag bool
+	stopResetFlag bool
```

Comment:
This name is confusing. resetFlag is quite obvious here.

Reply:
Yes but can't use the same global variable in both the files(`start.go` and `stop.go`). Therefore changed variable name according to the file.

Reply:
@kushthedude ?? how to proceed then?

Reply:
@thakursaurabh1998 @kushthedude it seems like moving the resetFlag to vars.go might make the most sense, yeah?

Reply:
ohh! yeah, I'll do it @leecalcote